### ISSUE TITLE
remove duplicate

### ISSUE THREAD

**Issue by Ski-ing** [nlpxucan/WizardLM#149]
Is there any strict operation to remove duplicate data between training data and test set human-eval before training?

**Comment by PoseidomWong** [comment]
I guess they didn't

**Comment by ChiYeungLaw** [comment]
We have checked the SFT training set. The HumanEval test set does not leak in it.

**Comment by PoseidomWong** [comment]
I would like to ask if there are any plans to open source the training data?
